modules/lightning_features/lightning_media/src/Plugin/EntityBrowser/Widget/FileUpload.php

```diff
@@ -104,8 +104,8 @@ public function submit(array &$element, array &$form, FormStateInterface $form_s
       $entity,
       MediaHelper::getSourceField($entity)->entity
     );
-    $file->setPermanent();
-    $file->save();
+    //$file->setPermanent();
```

Comment:
I can't merge this with unexplained, commented-out code in there. If this makes it work (and apparently it still passes the tests, so hey, alright!), then we need a comment here explaining why the file is not saved here.